cmd/contour/serve.go

```diff
@@ -1087,6 +1088,18 @@ func (s *Server) setupGatewayAPI(contourConfiguration contour_api_v1alpha1.Conto
 			}
 		}
 
+		// Create and register the TCPRoute controller with the manager.
+		if _, enabled := features["backendtlspolicy"]; enabled {
+			// Inform on ConfigMap if BackendTLSPolicy is enabled
+			if err := s.informOnResource(&corev1.ConfigMap{}, eventHandler); err != nil {
```

Comment:
I'm a little concerned about increased memory usage from the informer caching all ConfigMaps in the cluster. For Secrets, we did the following to avoid storing all the data from irrelevant Secrets: https://github.com/projectcontour/contour/blob/29201bbd13e5cf3477e35a8744f53bb13befb3b5/cmd/contour/serve.go#L253-L258

I think we should probably do something similar here for ConfigMaps since the vast majority of them will not be relevant for Contour. Probably checking for the `ca.crt` key, like you're doing in the DAG cache?

Reply:
Makes sense. Done.